ui/entry.jsx

```diff
@@ -1,6 +1,7 @@
 /* globals window, document */
 
 import './styles/base.less';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Use `bootstrap/..` directly.

Alternatively, you could have used `~/` instead of `../node_modules/`. `~` is a special alias in Webpack which, when used before a package name, will point to the correct location of the package in the `node_modules` hierarchy.